ca_tools/launch/draw_square.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+  <group ns="$(eval 'create' + str(arg('id')))" if="$(eval str(arg('id')).isdigit() and (arg('id') > 0))">
+    <arg name="id" default="$(optenv ID 1)" ns=/>
```

Comment:
Is this being used?

Reply:
Ok, so leave it commented and add a TODO message saying that you're going to add the feature later.